internal/dashboard/business/oceanbase/obcluster.go

```diff
@@ -543,28 +546,43 @@ func DeleteOBCluster(ctx context.Context, obclusterIdentity *param.K8sObjectIden
 	return oceanbase.DeleteOBCluster(ctx, obclusterIdentity.Namespace, obclusterIdentity.Name)
 }
 
-func GetOBClusterStatistic(ctx context.Context) ([]response.OBClusterStastistic, error) {
-	statisticResult := make([]response.OBClusterStastistic, 0)
+func GetOBClusterStatistic(ctx context.Context) ([]*response.OBClusterStastistic, error) {
```

Comment:
Why change the return type to a list of pointers

Reply:
No special reason. Merely unify the style the same as obtenants' statistic API.